Remotion/Web/Development.WebTesting.WebDriver/Web.Development.WebTesting.WebDriver.csproj

```diff
@@ -55,7 +55,6 @@
     <None Include="..\..\Remotion BSD.licenseheader">
       <Link>Remotion BSD.licenseheader</Link>
     </None>
-    <None Include="packages.config" />

```

Comment:
there's more changes required here.
* remove iedriver link 
* update nuspec file. 
* actually, we might get to completely remove the project since we no longer need it to package webdrivers.
* do not forget to update prjects.props.